File: UNIDAD 1/Tl1.4 RESTFUL_DOTNET_CONUNI_GR08/01.SERVIDOR/WS_ConUni_RESTFULDOTNET_GR08/ec.edu.monster.model/LoginModel.cs
namespace WS_ConUni_RESTFULDOTNET_GR08.ec.edu.monster.model
{
    public class LoginModel
    {
        public bool Login(string user, string password)
        {
            // Compara con el usuario y la contraseña hasheada
            return user == "MONSTER" && password == "MONSTER9";
        }
    }
}

File: ArquitecturaParcial1/UNIDAD 1/TI1.1 SOAP_JAVA_CONUNI_GR08/02.CLIESC/CLIESC_CONUNI_GR08/src/main/java/ec/edu/monster/modelo/LoginModel.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.modelo;
import ec.edu.monster.wsdl.Login;
import ec.edu.monster.wsdl.LoginService;
import ec.edu.monster.wsdl.Login_Service;

/**
 *
 * @author johan
 */
public class LoginModel {
    private final LoginService loginServicio;
    
    public LoginModel() {
        Login_Service loginService = new Login_Service();
        this.loginServicio = loginService.getLoginServicePort();
    }

    // Método para autenticar al usuario
    public boolean autenticar(String usuario, String contraseña) {
        try {
          
            return loginServicio.login(usuario, contraseña);
        } catch (Exception e) {
            throw new RuntimeException("Error al autenticar: " + e.getMessage());
        }
    }
    
}
